src/thriveopedia/fossilisation/FossilisationDialog.tscn

```diff
@@ -51,7 +47,11 @@ margin_right = 152.0
 [node name="Label" parent="VBoxContainer" index="0"]
 margin_right = 137.0
 
-[node name="HBoxContainer" parent="VBoxContainer" index="1"]
+[node name="FocusGrabber" parent="VBoxContainer" index="1"]
+margin_right = 137.0
+NodeToGiveFocusTo = NodePath("../../VBoxContainer2/HBoxContainer/CancelButton")
```

Comment:
This focus grabber doesn't work here because the normal confirmation buttons are all hidden.
Is this quick to fix or should I open a separate issue to no longer have the fossilisation dialog inherit the confirmation dialog (it should be a base custom dialog)?